code/modules/projectiles/guns/projectile.dm

```diff
@@ -165,4 +165,10 @@
 	desc = "A universal syndicate small-arms suppressor for maximum espionage."
 	icon = 'icons/obj/guns/projectile.dmi'
 	icon_state = "suppressor"
-	w_class = 2
\ No newline at end of file
+	w_class = 2
+
+ /obj/item/weapon/suppressor/specialoffer
+	name = "cheap suppressor"
+	desc = "A foreign knock-off suppressor, it feels flimsy, cheap, and brittle. Still fits all weapons."
```

Comment:
truly probably my most nitpicky thing, but the base one doesn't fit all guns either. Probably should say some.